docs/admin_guide.md

```diff
@@ -0,0 +1,97 @@
+# Admin Guide for Program & Events Dashboard
+
+This guide provides an overview of the Program & Events Dashboard. It also offers resources for managing and troubleshooting the system.
+
+## Table of Contents
+1. [Overview](#overview)
+2. [Monitoring and Logs](#monitoring-and-logs)
+   - [Toolforge](#toolforge)
+   - [Cloud VPS](#cloud-vps)
+3. [Troubleshooting](#troubleshooting)
+    - [Web Server Issues](#web-server-issues)
+    - [Database Issues](#database-issues)
+    - [Data Dumps and Recovery](#data-dumps-and-recovery)
+4. [More Resources](#more-resources)
+
+## Overview
+
+The **Programs & Events Dashboard** ([outreachdashboard.wmflabs.org](https://outreachdashboard.wmflabs.org)) is a web application designed to support the global Wikimedia community in organizing various programs, including edit-a-thons, education initiatives, and other events. **[Source Code](https://github.com/WikiEducationFoundation/WikiEduDashboard/tree/wmflabs)** **[Phabricator Project](https://phabricator.wikimedia.org/project/manage/1052/)**  
+
+### Infrastructure Overview
```

Comment:
This should include prose describing the different elements of the infrastructure.

Reply:
How is this?

## Infrastructure Overview
The application is hosted on the Wikimedia Cloud VPS project [globaleducation](https://openstack-browser.toolforge.org/project/globaleducation), which provides the infrastructure for all servers, allowing the dashboard to run on virtual machines that are flexible and easily managed within Wikimedia Cloud. 

The dashboard leverages the [wikiedudashboard](https://toolsadmin.wikimedia.org/tools/id/wikiedudashboard) Toolforge tool, a collection of PHP endpoints,  to retrieve revision and article data from Wikimedia Replica databases in the Wikimedia Cloud environment. The tool is deployed at [wikiedudashboard.toolforge.org](https://wikiedudashboard.toolforge.org/) and [replica-revision-tools.wmcloud.org](https://replica-revision-tools.wmcloud.org/). **Source code: [WikiEduDashboardTools](https://github.com/WikiEducationFoundation/WikiEduDashboardTools)**


### Servers

The dashboard operates on a distributed server architecture to handle web requests, process background jobs, and store application data. Each server is dedicated to specific roles, minimizing competition for resources and improving reliability by isolating potential bottlenecks and failures.

Below is a breakdown of the key servers and their roles within the infrastructure:

1. **Web Server**  ...........................|

```  ```
This currently describes toolforge, cloud vps and the 4 web servers, are there any other aspects I should mention?

Reply:
This is a little off. The WikiEduDashboardTools service is currently only used at replica-revision-tools.wmcloud.org.

We do also use wikiedudashboard.toolforge.org, but only as a source of static assets for the Vega.js files for Programs & Events Dashboard (see _vega.html.haml).

Reply:
There is also the reference-counter API: https://toolsadmin.wikimedia.org/tools/id/reference-counter (see reference_counter_api.rb).

Reply:
One more: https://ruby-suspected-plagiarism.toolforge.org/ - https://toolsadmin.wikimedia.org/tools/id/ruby-suspected-plagiarism

It's probably also worth including details about the non-WMF-production services/tools that parts of the system rely on, including:
* PetScan
* PagePile
* WikiWho
* Copypatrol (the database for which is used by the ruby-suspected-plagiarism tool)